### ISSUE TITLE
New results

### ISSUE THREAD

**Issue by jinshijian** [PNNL-TES/landmodels#11]
Hello @wwieder @bpbond @jianqiuz @kdorheim, I obtained Rh_annual from Warner, Tang, and Hashimoto based on the srdb-v5 sites' measure year, latitude, and longitude information, then I compared the measured Rh_annual with the modeled Rh_annual, please see figures below
* Violin plots show the mean, quantities, and distribution
![image](https://user-images.githubusercontent.com/13302161/82089246-4513c700-96c1-11ea-88b2-c03e225df5fc.png)
* Regression
![image](https://user-images.githubusercontent.com/13302161/82089258-4a711180-96c1-11ea-9baf-999cc156d51f.png)

Similarly, based on the MGRsD, I obtained Rh estimates from mimics, casa, and corpose, and compared the modeled results with the measured values, see figures below
* Violin plots show the mean, quantities, and distribution
![image](https://user-images.githubusercontent.com/13302161/82089327-6b396700-96c1-11ea-9c5b-ab2a2a289db8.png)
* Violin plots show the mean, quantities, and distribution, but separate by Season and month
![image](https://user-images.githubusercontent.com/13302161/82089333-6f658480-96c1-11ea-9bc0-3831e679ed70.png)
* Regression
![image](https://user-images.githubusercontent.com/13302161/82089338-72607500-96c1-11ea-9cc6-9fb8ca53c2b5.png)

There are definitely some very interesting results

**Comment by jinshijian** [comment]
The red dashed lines in the regression figures are 1:1 lines, and orange lines are linear regression lines. Rs should be Rh in the x-axis and y-axis labels.

**Comment by wwieder** [comment]
This looks interesting, @jinshijian?  Specifically I'm wondering if low or zero productivity grid cells are masked out of the modeled respiration fluxes (e.g. Sahara, Greenland and Antartica)?  

I'm not surprised that the testbed Rh results all look very similar, as they received the same input data forcing data.  This makes me wonder what we can say to evaluate the soil models or observational data products if the boundary conditions (potential differences in productivity, soil temperature, or soil moisture) provide the first order control over differences between observational data products and the simulated fluxes?

**Comment by bpbond** [comment]
@wwieder One answer, perhaps, is those plots between the three soil models themselves? For example, it looks to me like MIMICS does a better job predicting, or at least generating, high fluxes, which all three substantially under-predict relative to the site-specific data. (As do the Hashimoto, Tang, and Warner datasets--but as these are all statistical smoothings of the underlying observations, that's unsurprising.)

@jinshijian Nice! This site-specific test is a good one; because of scale mismatches, we wouldn't expect the models to actually predict sites' Rh, but in _aggregate_ it's troubling not to see better performance. I think coupling this with the latitudinal-band comparison will be illuminating.

**Comment by jianqiuz** [comment]
@jinshijian Cool study!  Just to make sure I get the context correctly: Warner, Tang, and Hashimoto are (climate) data-driven statistical models, while mimics, casa, and corpose are models with more mechanistic representation. I am curious about (1) Outlier analysis in the annual RH plot : are the observed high values associated with additional soil/climate/episodic factors? (2) In the daily rate prediction, the systematic model overestimation at low RH and underestimation at high RH to me is suggesting missing control in MIMICS, CORPSE and CASA. Are these models comparable in the way representing above ground processes, temperature and moisture sensitivity and microbial dynamics?

**Comment by bpbond** [comment]
Also we should loop in @kaizadp - that was an oversight, sorry!

**Comment by jinshijian** [comment]
@wwieder For your question, the comparison here is based on the annual soil respiration from SRDB-V5 (roughly 1,000 annual Rh observations), based on the latitude, longitude, and time information, I obtained annual Rh from Hashimoto, Tang, and Warner's global Rh, and do the comparison. For the daily Rh comparison is based on the daily time scale global soil respiration database, and I obtained Rh from MIMICS, CASA, and CORPOSE (about 10,000 daily Rh observations). So we do not need to mask the desert and Antartica.

**Comment by jinshijian** [comment]
@jianqiuz , you are right, Warner, Tang, and Hashimoto are (climate) data-driven statistical model results, while mimics, casa, and corpose are models with more mechanistic representation. (1) Outlier analysis will be interesting to do, but I removed all the measurements with manipulation, so only Rh measured from "normal" conditions were used. But need to do more search to find out. (2) I don't have any thoughts on this question, will discuss with Ben tomorrow.
